### PR TITLE
docs: Translate key-concepts

### LINKED ISSUE #813: Fill out translation for Concepts part of Backend.AI Core Documentation
### Main idea

There is a lot of missing documentation that needs translation, especially from English to Korean, Which brings a hard time to understand the documentation who cannot understand English very well (for Koreans).
Let's fill out the Korean translation for the Concepts part.
Here's the part that needs translations.
https://github.com/lablup/backend.ai/tree/main/docs/locales/ko/LC_MESSAGES/concepts

### Alternative ideas

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by chichoon**
close #813

- Translated Manager and Agents
- Translated Sessions and Kernels
- Translated Cluster Networking and Databases
- Translated Configuration Management
- Translated Virtual Folders

@lizable 
@kyujin-cho

### CODE REVIEW

File: docs/locales/ko/LC_MESSAGES/concepts/api-overview.po [hunk 1]
```diff
@@ -20,20 +20,20 @@ msgstr ""
...
 #: ../../concepts/api-overview.rst:2 d402740be3654961964d10c75cc75beb
 msgid "API Overview"
-msgstr "API 개괄"
+msgstr "API 개요"
 
 #: ../../concepts/api-overview.rst:4 c41cae658e2d49bda5ee7e4207d959f5
 msgid "Backend.AI API v3 consists of two parts: User APIs and Admin APIs."
 msgstr ""
-"Backend.AI API v3는 두 가지로 구성 되어있습니다: 사용자 API와 관리자 API."
+"Backend.AI API v3는 사용자 API와 관리자 API의 두 종류로 구성되어 있습니다."
```
Comment (lizable) [inline docs/locales/ko/LC_MESSAGES/concepts/api-overview.po]: I think we don't need to add `의` in here.

**Review by lizable — CHANGES_REQUESTED** [review]
@chichoon 
Please check the review msg and apply the changes.

File: docs/locales/ko/LC_MESSAGES/concepts/api-overview.po (hunk 1 above)
Comment (chichoon) [reply docs/locales/ko/LC_MESSAGES/concepts/api-overview.po]: ok

**Review by lizable — APPROVED** [review]
LGTM!

**Pull request merged**